ui/editor.go

```diff
@@ -0,0 +1,22 @@
+package ui
+
+import (
+	"os"
+	"os/exec"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+type editorFinishedMsg struct{ err error }
+
+func openEditor(path string) tea.Cmd {
+	editor := os.Getenv("EDITOR")
+	if editor == "" {
+		editor = "vim"
```

Comment:
Maybe it's better to change to nano as default? It's very likely that nano is pre-installed on OS, very unlikely that vim will be present.

Reply:
That's a good point, will change. If someone uses `vim` they should probably have their `$EDITOR` set as well 👍